src/test/scala/nl/knaw/dans/easy/multideposit/actions/AddFileMetadataToDepositSpec.scala

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
General note about tests: make sure that the lines printed during the build give useful information. These tests now output:

		AddFileMetadataToDepositSpec:
		checkPreconditions
		- should always succeed
		run
		- should write the file metadata to an xml file
		rollback
		- should always succeed
		datasetToFileXml
		- should produce the xml for all the files
		xmlPerPath
		- should produce the xml for one file

So, in two cases it only says `should always succeed`. This is of course because in the code it continues with `in {...}`. To make the test results during the build easier to navigate, please add a more descriptive part in the test title.